types/index.d.ts

```diff
@@ -2,3 +2,6 @@
 
 import DayPicker from './DayPicker';
 export default DayPicker;
+
```

Comment:
if you add `export * from './DayPickerInput';` it'll also solve https://github.com/gpbl/react-day-picker/issues/586

Reply:
that is semantically incorrect. DayPickerInput is not exported from the root of the package: https://github.com/gpbl/react-day-picker/blob/master/DayPicker.js

Reply:
Hm, what would be the correct way then? Kinda new to typescript.

Reply:
since the JS export is from a submodule, you can import in typescript like this:

```ts
import { DayPickerInput } from "react-day-picker/DayPickerInput";
```

that's why [`types/DayPickerInput.d.ts`](https://github.com/gpbl/react-day-picker/blob/master/types/DayPickerInput.d.ts) exists.

Reply:
Importing like that doesn't work, my compiler complains about no declaration file found.

Reply:
hm, ok, can you file an issue for that? I might be able to look into it later

Reply:
Should I open up a new issue or is https://github.com/gpbl/react-day-picker/issues/586 good enough? Thank you!